dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/fat/src/io/openliberty/microprofile/telemetry/internal/utils/jaeger/JaegerQueryClient.java

```diff
@@ -63,7 +63,7 @@ public class JaegerQueryClient implements AutoCloseable {
     public JaegerQueryClient(JaegerContainer container) {
         this(container.getHost(), container.getQueryGrpcPort());
     }
-
+    
     public JaegerQueryClient(String host, int port) {
```

Comment:
Try to avoid making non-changes like this. If you squash the commits in this PR, please remove the change to this file and to `JaegerContainer`, `ZipkinQueryClient` and `ZipkinSpan`.